src/Uno.UI.Wasm/ts/Windows/Devices/Midi/Internal/WasmMidiAccess.ts

```diff
@@ -0,0 +1,28 @@
+﻿namespace Uno.Devices.Midi.Internal {
+	export class WasmMidiAccess {
+		private static midiAccess: WebMidi.MIDIAccess;
+
+		private static dispatchRequest: (hasAccess: boolean) => number;
+
+		public static request(): Promise<string> {			
+			if (navigator.requestMIDIAccess) {
+				console.log('This browser supports WebMIDI!');
+				return navigator.requestMIDIAccess()
+					.then(
+						(midi: WebMidi.MIDIAccess) => {
+							WasmMidiAccess.midiAccess = midi;
+							return "true";
+						},
+						() => "false");
+			}
+			else {
+				console.log('WebMIDI is not supported in this browser.');
+				return Promise.resolve("false");
+			}
+		}
+
+		public static getMidi(): WebMidi.MIDIAccess {			
+			return this.midiAccess;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the parent class name instead of `this` when in a `static` context.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)